### DISCUSSION TITLE
Package front-end not loading for huge tables/pre

### DISCUSSION THREAD

**Discussion by llrs** [r-universe-org/help#244] [Q&A] [answered]
I am trying to visualize a package in a universe: https://ropenspain.r-universe.dev/BOE#
But I'm getting problems from multiple browsers, the page starts loading but then it turns in a blank webpage.

The network tool show 2 errors in Chromium:

```
Failed to load resource: net::ERR_BLOCKED_BY_CLIENT https://r-universe.dev/matomo.js
Failed to load resource: the server responded with a status of 400 () https://ropenspain.r-universe.dev/shared/scienceminer/BOE
```

I'm not sure if they are related but disabling ad-blockers didn't fix the issue, so I think it is related to the "No ScienceMiner data found BOE" log I see.

**Comment by jeroen** [comment]
I am aware of this bug. It is caused by overly wide code blocks or tables in the readme, which makes the content div overflow and get pushed to the bottom somehow (if you scroll all the way down that is where the page is).

I'll try to look into it again, but you can avoid the problem by giving the knitr output in the readme linebreaks.

**Reply by llrs** [reply]
Oh, I didn't realize I could scroll. I'll try to add linebreaks. 
I am surprised this is the cause as pkgdown handles this wide tables without problems (AFAIK). 
Thanks!

**Comment by jeroen** [comment]
I think this fixes it: https://github.com/r-universe-org/dashboard/commit/727553e65bacf8c1ee8f8bde1cb270e6d1910014

**Comment by jeroen** [comment]
There is still a similar problem for huge tables though :/

**Comment by jeroen** [comment, marked as answer]
OK tables solved now too. Yay.
